src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -294,6 +295,10 @@ public static void tenantOp(String tenantId, JsonObject job) throws InterruptedE
       failureMessage = String.format("Tenant get failed: %s: %s",
           response.getStatusCode(), response.getBody());
 
+      if (response.getJson().containsKey("error")) {
+        throw new IllegalStateException(response.getJson().getString("error"));
+      }
```

Comment:
Is the response body always JSON?

If not, could `response.getJson()` throw an exception?

Reply:
I've added additional status check, for 200 it is always JSON.